src/ocean/text/arguments/TimeIntervalArgs.d

```diff
@@ -356,7 +382,7 @@ unittest
     interval = processTimeIntervalArgs(args);
 
     Test.test!("==")(interval.begin, TEST_TIME_NOW - 60);
-    Test.test!("==")(interval.end, TEST_TIME_NOW + 59);
+    Test.test!("==")(interval.end, TEST_TIME_NOW - 1);
```

Comment:
This commit contains two changes: 1. moving the duplicate part out 2. functionality change, where in case of 2 arguments `end` becomes relating to `begin` and not relative to `now`. Due to the functionality change you had to update the unit test.  

So, you could split this commit to two: the 1st commit would contain the unit test update `Test.test!("==")(interval.end, TEST_TIME_NOW - 1);` and the functionality change:
```
    begin = time(null) - parseTimeInterval(begin_arg);	
    end = begin + parseTimeInterval(end_arg);
```

Instead of the previous:
```
    begin = time(null) - parseTimeInterval(begin_arg);	
    end = time(null) + parseTimeInterval(end_arg);
```

The 2nd commit can contain the rest.